### PR TITLE
add Axe testing

### PR DESCRIPTION
**Pull request by BrianHicks**
This adds automatic accessibility testing to the styleguide. Right now we have 155 failures! So I've made it so that the test will pass if we have exactly that many, and fail if we go any higher or lower (so that we do not forget to lower the budget.)

### CODE REVIEW

File: script/format-axe-report.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# failures to the correct value.
+NUM_ERRORS="$(jq '.violations | map(.nodes | length) | add' "$JSON_FILE")"
+TARGET_ERRORS=155
+if test "$NUM_ERRORS" -ne "$TARGET_ERRORS"; then
+    echo "got $NUM_ERRORS errors, but expected $TARGET_ERRORS."
+    echo
+    echo 'If it went down, hooray!'
+    echo "Check out ${0:-} and change the count to the reported value above."
+    echo
+    echo "Did it go up? That's OK too."
```
Comment (tesk9) [inline script/format-axe-report.sh]: I read this as "feel free to increase the violations count." Is that what you mean?

**Comment by tesk9** [comment]
(Also, I am EXTREMELY enthused about this! THANK YOU!)

**Comment by BrianHicks** [comment]
oh! I didn't add it to the test target. That's what's going on. I tested by removing some low-hanging fruit (duplicate IDs in embedded SVGs) and it gave an error.

**Comment by BrianHicks** [comment]
oh sorry, that is to say: give it another go, `make test` should do the right thing now.

File: script/format-axe-report.sh (hunk 1 above)
Comment (BrianHicks) [reply script/format-axe-report.sh]: nope! reworded.

**Pull request merged**